settings/blt.settings.php

```diff
@@ -70,6 +71,7 @@
  * @var $site_path
  * This is always set and exposed by the Drupal Kernel.
  */
+// phpcs:ignore
```

Comment:
I can't immediately tell what's the ignored problem here.

Reply:
`$site_path` is a global variable, but PHPCS doesn't know that so it thinks it's undefined. We can't simply declare it as global either, because it's named inappropriately (not our fault as mentioned in the doc comment, it comes from Drupal core). I don't know of anything else to do besides ignore it.

Reply:
Do note that [you can limit only specific sniffs](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file) to make the impetus clearer, e.g.:
```php
// phpcs:disable Generic.Commenting.Todo.Found
$xmlPackage = new XMLPackage;
$xmlPackage['error_code'] = get_default_error_code_value();
// TODO: Add an error message here.
$xmlPackage->send();
// phpcs:enable
```

Reply:
Thanks, it's a little more verbose and pretty much all of the ignores are for the same two reasons (global vars or shell exec) so I'll probably document it in the follow-up ticket instead.